appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/transitionmodel/promoter/interpolator/PromoterInterpolator.kt

```diff
@@ -36,6 +36,9 @@ class PromoterInterpolator<NavTarget : Any>(
     private val halfHeightDp = (transitionBounds.heightDp.value - childSize.value) / 2
     private val radiusDp = min(halfWidthDp, halfHeightDp) * 1.5f
 
+    override val finishedAnimations: Flow<NavElement<NavTarget>>
+        get() = TODO("Not yet implemented")
```

Comment:
TODO?

Reply:
it needs to be migrated to BaseInterpolator